grpc-mqtt.cabal

```diff
@@ -9,7 +9,10 @@ author:      Arista Networks
 maintainer:  opensource@awakesecurity.com
 copyright:   2021-2022 Arista Networks
 category:    Network
-build-type:  Simple
+build-type:  Custom
+
+custom-setup
+  setup-depends: base, Cabal, filepath
```

Comment:
This needs to come after the other top-level directives (extra-source-files, tested-with).

Reply:
Ah, OK.  Put just before "common" by 89c11c99d9f5bc62dce2fdcb542740ed5f8ece1b .